wwwroot/init/terria-cube.json

```diff
@@ -136,211 +132,1537 @@
+                "http://bom.gov.au/waterdata/services/stations/415207",
+                "http://bom.gov.au/waterdata/services/stations/415226",
+                "http://bom.gov.au/waterdata/services/stations/415237",
+                "http://bom.gov.au/waterdata/services/stations/224214A",
+                "http://bom.gov.au/waterdata/services/stations/604053",
+                "http://bom.gov.au/waterdata/services/stations/606001",
+                "http://bom.gov.au/waterdata/services/stations/606002",
+                "http://bom.gov.au/waterdata/services/stations/606185",
+                "http://bom.gov.au/waterdata/services/stations/607155",
+                "http://bom.gov.au/waterdata/services/stations/608002",
+                "http://bom.gov.au/waterdata/services/stations/610008",
+                "http://bom.gov.au/waterdata/services/stations/613002",
+                "http://bom.gov.au/waterdata/services/stations/613146",
+                "http://bom.gov.au/waterdata/services/stations/614044",
+                "http://bom.gov.au/waterdata/services/stations/616002",
+                "http://bom.gov.au/waterdata/services/stations/616013",
+                "http://bom.gov.au/waterdata/services/stations/616065",
+                "http://bom.gov.au/waterdata/services/stations/803003",
+                "http://bom.gov.au/waterdata/services/stations/804001",
+                "http://bom.gov.au/waterdata/services/stations/809310"
+              ],
+              "procedures": [
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/tstypes/Pat3_C_B_1_YearlyMean",
+                  "title": "Annual average",
+                  "defaultDuration": "40y"
+                },
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/tstypes/Pat3_C_B_1_MonthlyMean",
+                  "title": "Monthly average",
+                  "defaultDuration": "3y"
+                },
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/tstypes/Pat3_C_B_1_DailyMean",
+                  "title": "Daily average"
+                },
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/tstypes/Pat3_C_B_1_DailyMin",
+                  "title": "Daily minimum"
+                },
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/tstypes/Pat3_C_B_1_DailyMax",
+                  "title": "Daily maximum"
+                },
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/tstypes/Pat3_C_B_1_HourlyMean",
+                  "title": "Hourly average",
+                  "defaultDuration": "48h"
+                }
+              ],
+              "observableProperties": [
+                {
+                  "identifier": "http://bom.gov.au/waterdata/services/parameters/Water Course Level",
```

Comment:
Could you please make sure urls are ASCII encoded and replace the spaces here with `%20`? It should theoretically work correctly already, but it leaves less to chance. 😄

Reply:
@reginapramesti found this breaks the links, never mind 😬